src/distances.jl

```diff
@@ -24,53 +32,76 @@ julia> Ellipsoidal([1.0,0.5], [π/2])
 julia> Ellipsoidal([1.0,0.5,0.5], [π/2,0.0,0.0])
 ```
 """
+
+# Right-hand rule; motion defined looking to the negative direction of the axis
+struct RotationRules
```

Comment:
Perhaps rename the struct to singular `RotationRule` given that it only stores a single rule.

Reply:
Done